### PR TITLE
znk chat

### PR DESCRIPTION
**Pull request by kostamas**

### CODE REVIEW

File: demo/znkChat/demo.js [hunk 1]
```diff
@@ -54,7 +57,7 @@
         fbDataEndPoint: "https://sat-dev.firebaseio.com/",
         appContext: 'student',
         studentAppName: 'sat_app',
-        dashboardAppName: 'sat_dashboard',
+        firebaseDashboardAppScopeName: 'sat_dashboard',
```
Comment (assafshp) [inline demo/znkChat/demo.js]: are you sure we have such a key?

File: src/components/znkChat/services/znkChatSrv.service.js [hunk 2]
```diff
@@ -113,6 +114,8 @@
                     var secondUserWriteChatGuidsProm = chatterRef.update(userNewChatGuid);
                     return $q.all([localUserWriteChatGuidsProm, secondUserWriteChatGuidsProm]).then(function () {
                         return chatGuid;
+                    },function(error){
+                        $log.error('znkChat- create new chat: ' + error);
```
Comment (assafshp) [inline src/components/znkChat/services/znkChatSrv.service.js]: I prefer - 'znkChat: error while creating new chat: "

File: src/components/znkChat/services/znkChatSrv.service.js [hunk 3]
```diff
@@ -85,6 +85,7 @@
                         }
                     }
                 }
+                return undefined;
```
Comment (assafshp) [inline src/components/znkChat/services/znkChatSrv.service.js]: why

File: demo/znkChat/demo.js (hunk 1 above)
Comment (kostamas) [inline demo/znkChat/demo.js]: yes, i copied that key from act student app ENV

File: src/components/znkChat/services/znkChatSrv.service.js (hunk 3 above)
Comment (kostamas) [inline src/components/znkChat/services/znkChatSrv.service.js]: to insure that the function return undefined, but yest it's not necessary

**Pull request merged**